lib/index.js

```diff
@@ -1,126 +1 @@
-  }
-
-  if (!args) {
-    throw new AuthSdkError('No arguments passed to constructor. ' +
-      'Required usage: new OktaAuth(args)');
-  }
-
-  if (!args.url) {
-    throw new AuthSdkError('No url passed to constructor. ' +
-      'Required usage: new OktaAuth({url: "https://sample.okta.com"})');
-  }
-
-  this.options = {
-    url: args.url,
-    clientId: args.clientId,
-    redirectUri: args.redirectUri,
-    ajaxRequest: args.ajaxRequest || ajaxRequest,
-    transformErrorXHR: args.transformErrorXHR,
-    headers: args.headers
-  };
-
-  // Remove trailing forward slash from url
-  if (this.options.url.slice(-1) === '/') {
-    this.options.url = this.options.url.slice(0, -1);
-  }
-
-  sdk.session = {
-    close: util.bind(session.closeSession, sdk, sdk),
-    exists: util.bind(session.sessionExists, sdk, sdk),
-    get: util.bind(session.getSession, sdk, sdk),
-    refresh: util.bind(session.refreshSession, sdk, sdk),
-    setCookieAndRedirect: util.bind(session.setCookieAndRedirect, sdk, sdk)
-  };
-
-  sdk.tx = {
-    status: util.bind(tx.transactionStatus, sdk, sdk),
-    resume: util.bind(tx.resumeTransaction, sdk, sdk),
-    exists: util.bind(tx.transactionExists, sdk, sdk)
-  };
-
-  // This is exposed so we can mock document.cookie in our tests
-  sdk.tx.exists._getCookie = function(name) {
-    return cookies.getCookie(name);
-  };
-
-  sdk.idToken = {
-    authorize: util.bind(token.getIdToken, sdk, sdk),
-    verify: util.bind(token.verifyIdToken, sdk, sdk),
-    refresh: util.bind(token.refreshIdToken, sdk, sdk),
-    decode: util.bind(token.decodeIdToken, sdk)
-  };
-
-  // This is exposed so we can mock window.location.href in our tests
-  sdk.idToken.authorize._getLocationHref = function() {
-    return window.location.href;
-  };
-}
-
-var proto = OktaAuth.prototype;
-
-proto.features = {};
-
-proto.features.isPopupPostMessageSupported = function() {
-  var isIE8or9 = document.documentMode && document.documentMode < 10;
-  if (window.postMessage && !isIE8or9) {
-    return true;
-  }
-  return false;
-};
-
-proto.features.isTokenVerifySupported = function() {
-  return typeof crypto !== 'undefined' && crypto.subtle && typeof Uint8Array !== 'undefined';
-};
-
-// { username, password, (relayState), (context) }
-proto.signIn = function (opts) {
-  return tx.postToTransaction(this, '/api/v1/authn', opts);
-};
-
-proto.signOut = function () {
-  return this.session.close();
-};
-
-// { username, (relayState) }
-proto.forgotPassword = function (opts) {
-  return tx.postToTransaction(this, '/api/v1/authn/recovery/password', opts);
-};
-
-// { username, (relayState) }
-proto.unlockAccount = function (opts) {
-  return tx.postToTransaction(this, '/api/v1/authn/recovery/unlock', opts);
-};
-
-// { recoveryToken }
-proto.verifyRecoveryToken = function (opts) {
-  return tx.postToTransaction(this, '/api/v1/authn/recovery/token', opts);
-};
-
-module.exports = OktaAuth;
+module.exports = require('../reqwest');
```

Comment:
Let's add a comment here about what this file is for (namely, that the default is to export a version of AuthJs that uses reqwest)